README.md

```diff
@@ -238,7 +241,84 @@ govuk-docker prune
 
 You should then be able to `govuk-docker build` your service and have confidence you're not suffering from configuration drift.
 
-### How to: set up Dnsmasq manually
+### How to: resolve setup issues
+
+The following sections resolve common installation issues.
+
+#### How to: update Ruby
+
+Update Ruby using [brew](https://brew.sh/), then add it to the PATH in your bash config.
+
+```
+brew install ruby
```

Comment:
The preferred way is to use `rbenv` to manage Ruby versions. With `rbenv`, an `rbenv install` in the root of the repo will install the version of Ruby specified in [`.ruby-version`](https://github.com/alphagov/govuk-docker/blob/master/.ruby-version), which is probably different (ie older) than the latest provided by Homebrew.